applications/luci-app-babeld/luasrc/model/cbi/babeld.lua

```diff
@@ -0,0 +1,126 @@
+m = Map("babeld", "Babeld Overview", translate("Babeld Overview"))
+m.pageaction = false
+
+s = m:section(NamedSection, "__babeldoverview__")
+
+function s.render(self, sid)
+	local tpl = require "luci.template"
+
+	tpl.render_string([[
+		<%
+		local utl = require "luci.util"
+		local v6_port = 33123
+			local dump = utl.exec("(echo dump | nc ::1 %d)" % v6_port)
```

Comment:
There the indentation does not fit.
I believe that the echo before dump is not necessary.
Is the command really called dump?

Reply:
Is the command really called dump?
 
 Yes. ;)